src/calculations.h

```diff
@@ -44,7 +44,7 @@ typedef struct CalculateCo2Parameters {
   const int kFuelEfficiency;
 } CalculateCo2Parameters;
 
-double CalculateDistance(
+int CalculateDistance(
```

Comment:
Why int?

Reply:
Actually dont know. I have changed it back. My bad